rabbitmq.py

```
@@ -0,0 +1,684 @@
+                'action_type': message.get('action_type'),
+                'status': message.get('status'),
+                'timestamp': message.get('timestamp'),
+                'details': message.get('details', {})
+            }
+            
+            # Send to all connected WebSocket clients
+            disconnected_clients = set()
+            for client in self.websocket_clients:
+                try:
+                    asyncio.create_task(client.send(json.dumps(websocket_message)))
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)